source/creating_content/01_creating_content_page/managing_page_security.rst

```diff
@@ -0,0 +1,118 @@
+Managing the security of a page
+===============================
+
+To manage a content page security, go to the Pages section of the back office, in the Security tab of the chosen content page.
+
+.. admonition:: Important
+   :class: important
+
+   General permissions at domain level have priority over permissions at page level. For example, if a user has the *Browse all pages* permission, they can access any page, even the private ones. They don't need to be added in the page and granted security permissions for that private page.
+
+Choosing an access policy
+-------------------------
+
+Content pages created with the OpenDataSoft platform can either be accessible by anyone allowed on the portal, or restricted to specifically defined users.
+
+To choose the access policy of a content page:
+
+1. Go to Security > Access policy.
+2. Click
+
+  - either, the **Public access** button, to let any user access the page, as long as they have access to the portal,
+  - or, the **Private access** button, to restrict the access to the page to specically defined users and/or groups of users.
+
+.. admonition:: Caution
+   :class: caution
+
+   If a user has access to a content page but which was created with datasets that user isn't allowed to access, all related visualizations will not load. However, the user will be able to read all other information displayed on the content page, such as text.
```

Comment:
I had to read this a few times to understand it. Very important point but I struggled with the structure of the sentences.